agent/ansible/README

```diff
@@ -22,5 +22,65 @@ file for guidance.
 
 * Usage
 #+begin_src shell
-ansible-playbook -i /path/to/inventory/pbench-agent.hosts pbench-agent-install.yml
+ansible-playbook -i /path/to/inventory/pbench-agent.hosts pbench_agent_install.yml
 #+end_src
+
+
+* Installing roles from Ansible Galaxy
+
+The roles that the above playbooks use are available from Ansible
+Galaxy, as part of the pbench.agent collection. You can install the
+collection with
+
+   ansible-galaxy collection install pbench.agent
+
+The roles will be installed under
+
+    $HOME/.ansible/collections/ansible_collections/pbench/agent
+
+and you will need to add this path to Ansible's roles path in order
+for the playbooks to find them:
+
+    ANSIBLE_ROLES_PATH=$HOME/.ansible/collections/ansible_collections/pbench/agent/roles:$ANSIBLE_ROLES_PATH
+    export ANSIBLE_ROLES_PATH
+
+or, more permanently, you can add it to the /etc/ansible/ansible.cfg
+file:
+
+You will still need a playbook to install the pbench agent. You can use the
+standard pbench_agent_install.yml from the pbench repo - here's a link:
+
+    https://github.com/distributed-system-analysis/pbench/blob/master/agent/ansible/pbench-agent-install.yml
```

Comment:
Looks like you missed the dash-to-underscore (dash_to_underscore?) change here, and at 57.

Reply:
Old habits die hard...